rspack.main.config.js

```diff
@@ -3,15 +3,22 @@
 /* eslint-disable import/no-commonjs */
 const fs = require("fs");
 
-const ReactRefreshPlugin = require("@pmmmwh/react-refresh-webpack-plugin");
+const rspack = require("@rspack/core");
+const ReactRefreshPlugin = require("@rspack/plugin-react-refresh");
 const HtmlWebpackPlugin = require("html-webpack-plugin");
-const MiniCssExtractPlugin = require("mini-css-extract-plugin");
 const NodePolyfillPlugin = require("node-polyfill-webpack-plugin");
-const path = require("path");
-const TerserPlugin = require("terser-webpack-plugin");
-const webpack = require("webpack");
 const WebpackNotifierPlugin = require("webpack-notifier");
 
+const {
```

Comment:
The diff in this file is weird. Actually i:
- renamed `rspack.config.js` to `rspack.main.config.js`
- removed `sub`-configs from it
- added usage of some constants from `frontend/build`:
```
const {
  IS_DEV_MODE,
  LICENSE_TEXT,
} = require("./frontend/build/shared/constants");
const { BABEL_CONFIG } = require("./frontend/build/shared/rspack/babel-config");
const { CSS_CONFIG } = require("./frontend/build/shared/rspack/css-config");
const {
  getBannerOptions,
} = require
```

all other things are untouched